src/apps/backend/modules/account/internal/account-reader.ts

```diff
@@ -54,18 +53,6 @@ export default class AccountReader {
 
   public static async getAccountByPhoneNumber(
     phoneNumber: PhoneNumber
```

Comment:
I am not really expecting a change in backend, would like to understand why its needed